pepper-apis/src/main/java/org/broadinstitute/ddp/copy/AnswerToProfileCopier.java

```diff
@@ -34,7 +35,7 @@ public AnswerToProfileCopier(Handle handle, long operatorId) {
         this.handle = handle;
         this.operatorId = operatorId;
         this.jdbiCompositeQuestion = handle.attach(JdbiCompositeQuestion.class);
-        this.profileDao = handle.attach(UserProfileDao.class);
+        this.profileDao = new UserProfileCachedDao(handle);
```

Comment:
Here we go. AnswerToProfileCopier actually modifies User data. And if we change it, we don't want to keep caching data that is not up to date. The UserProfileCachedDao notifies CacheService that user data has been modified and CacheService finds the dependent caches and evicts the data.
Tried to make the code change easy here.